src/software/ai/hl/stp/play/stop_play.cpp

```diff
@@ -108,7 +108,8 @@ void StopPlay::getNextTactics(TacticCoroutine::push_type &yield)
         // line from the center of the goal to the ball intersects. A robot will be placed
         // on that line, and the other two will be on either side
         Point ball_defense_point_center =
-            world.ball().position() + 0.5 * goal_to_ball_unit_vector;
+            world.ball().position() +
+            (0.5 + 2 * ROBOT_MAX_RADIUS_METERS) * goal_to_ball_unit_vector;
```

Comment:
Can you update the comment above to explain the change?

Reply:
Wouldn't we want to 1 robot radius away from the circle we don't want to be in? Is there an additional robot radius buffer?

Reply:
Yea my assumption is we wanted an extra buffer because we were sitting too close and were at risk of getting a foul. Updated comment.